daemon/daemon.go

```diff
@@ -101,7 +103,16 @@ func (c *Command) canAccess(r *http.Request, user *auth.UserState) bool {
 		}
 
 		if c.PolkitOK != "" {
-			if authorized, err := polkitCheckAuthorizationForPid(pid, c.PolkitOK, nil, polkit.CheckAllowInteraction); err == nil {
+			allow, err := strconv.ParseBool(r.Header.Get(client.AllowInteractionHeader))
```

Comment:
This seems to be failing and reporting an error on an empty header. It should instead silently use the default behavior.